httpgrpc/protocol_versions.go

```diff
@@ -29,4 +31,19 @@ package httpgrpc
 const (
 	UnaryRpcContentType_V1  = "application/x-protobuf"
 	StreamRpcContentType_V1 = "application/x-httpgrpc-proto+v1"
+
+	GrpcWebJsonContentType_V1 = "application/grpc-web+json"
+
+	ApplicationJson = "application/json"
+)
+
+var (
+	jsonMarshaler = jsonpb.Marshaler{
```

Comment:
It's a shame that these are hard-coded.

In an ideal world, we would allow the calling application to configure these properties and just use a zero-value marshaler and unmarshaler by default.

Perhaps we don't need to upstream these changes until we have something that is a bit more polished and configurable? I worry that adding these now will mean we have to _keep_ things this way and limits our ability to evolve this, due to backwards compatibility issues.

Reply:
Fair points!  We could just leave this open / convert it to an issue.